tests/EnviroSense.Application.Tests/Authorization/AccessRules/ApiKeyAccessRuleTest.cs

```diff
@@ -73,23 +73,24 @@ private Guid SampleGuid()
         return Guid.Parse("5fe7be6c-4ce6-43ce-94f5-e4f91df55a74");
     }
 
-    private DeviceApiKey SampleDeviceApiKey()
+    private ApiKey SampleDeviceApiKeys()
```

Comment:
```suggestion
    private ApiKey SampleApiKey()
```

Suggested name was not applied 😏